docs/guides/deployment/overriding-job-variables.md

```diff
@@ -0,0 +1,176 @@
+# Deeper Dive: Overriding Work Pool Job Variables
+
+As you might have read in [Deploying Flows to Work Pools and Workers](/guides/prefect-deploy/) guide, there are two ways to deploy flows to work pools: using a `prefect.yaml` file or using the `.deploy()` method.
+
+**In both cases, you can override job variables on a work pool for a given deployment.**
+
+While exactly _which_ job variables are available to be overridden depend on the type of work pool you're using at a given time, this guide will explore some common patterns for overriding job variables in both deployment methods.
+
+## Background
+First of all, what do we mean by _"job variables"_?
+
+Job variables are just infrastructure related values that are configurable on a work pool, which may be relevant to how your flow run executes on your infrastructure.
+
+Let's use `env` as an example, because it's the only job variable that is available to configure for all work pool types.
```

Comment:
vertical break
```suggestion
<hr>

Let's use `env` as an example, because it's the only job variable that is available to configure for all work pool types.
```